controllers/positions.js

```diff
@@ -0,0 +1,95 @@
+const fs = require('fs');
+const positions =  require('../data/positions.json');
+
+//Getting all job positions
+const getAllPositions = (req, res) => {
+    return res.json(positions)
+};
+//Getting a single job position
+const getPositionsByJobTitle= (req, res) => {
+    let filtPositions= positions.filter(item => item.job_title.includes(req.params.job_title));
+    return res.json(filtPositions);
+};
+//Creating a job position
+const createJobPosition = (req, res) => {
+    const newPosition = {
+        id: (positions.length)+1,
```

Comment:
There's no need to use the `()` in this case. You can do just something like:

```suggestion
        id: positions.length + 1,
```